create_a_materialized_view_batch_ingest/.tinyenv

```diff
@@ -1 +1 @@
-VERSION=0.0.0
+VERSION=0.0.1
```

Comment:
Bumping the minor auto-promotes to "live", in this case where there is a populate, I recommend to deploy in a "preview" release and then, after checking everything is ok, go to "live". For that, bump the major 0.0.1->1.0.0